app/decorators/link_decorator.rb

```diff
@@ -21,8 +21,8 @@ def format_expiration
   def render_remove_link
     if h.current_user.id == user_id ||
        h.current_user.position == Position.find_by(name: 'Secretary')
-      h.button_to('remove', h.deactivate_link_path(link),
-                  class: 'btn btn-xs btn-danger')
+      h.link_to('remove', h.deactivate_link_path(link), method: :post,
+                 class: 'btn btn-xs btn-danger')
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.